atlas-mongodb-mixin/README.md

```diff
@@ -0,0 +1,103 @@
+# MongoDB Atlas Mixin
+The MongoDB Atlas mixin is a set of configurable Grafana dashboards and alerts.
+
+The MongoDB Atlas  mixin contains the following dashboards:
+
+- Atlas cluster overview
+- Atlas performance overview
+- Atlas operations overview
+- Atlas elections overview
+- Atlas sharding overview
+
+and the following alerts:
+
+- AtlasHighNumberOfDeadlocks
```

Comment:
I think these alerts Should be `MongoDBAtlas` prefixed (linter considered)

Atlas just seems like it could be another technology not related to MongoDB.